test/serviceaccount/secret.yaml

```diff
@@ -15,6 +15,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: test-readonly-credentials
+# type: kubernetes.io/dockerconfigjson
```

Comment:
Why comment this out?

Reply:
For testing originally, i had a setup that only worked with the dockerconfigjson format so I modified this secret to be of that type for my tests.  Now it works for either type (dockercfg or dockerconfigjson), I will remove these comments.

Reply:
done